### ISSUE TITLE
dosnt work with ustream from iss

### ISSUE THREAD

**Issue by treaki** [rg3/youtube-dl#3760]
hi,

i tryed to watch this stream without flash:

http://www.ustream.tv/channel/iss-hdev-payload

but i only got this:

```
$ youtube-dl --verbose "http://www.ustream.tv/channel/iss-hdev-payload"
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://www.ustream.tv/channel/iss-hdev-payload']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2014.09.15.1
[debug] Python version 2.6.6 - Linux-2.6.32-5-686-bigmem-i686-with-debian-6.0.9
[debug] Proxy map: {}
[ustream:channel] iss-hdev-payload: Downloading webpage
[ustream:channel] iss-hdev-payload: Downloading video information (next: 1)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 15)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 28)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 41)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 54)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 66)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 81)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 96)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 108)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 121)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 137)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 149)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 161)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 174)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 186)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 198)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 210)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 224)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 236)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 248)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 260)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 273)
[ustream:channel] iss-hdev-payload: Downloading video information (next: 286)
[download] Downloading playlist: 17074538
[ustream:channel] playlist 17074538: Collected 287 video ids (downloading 287 of them)
[download] Downloading video #1 of 287
[ustream] 52711541: Downloading webpage
[ustream] 52711541: Extracting information
[debug] Invoking downloader on u'http://tcdn.ustream.tv/video/52711541'
ERROR: unable to download video data: HTTP Error 404: Not Found
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1035, in process_info
    success = dl(filename, info_dict)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1011, in dl
    return fd.download(name, info)
  File "/usr/local/bin/youtube-dl/youtube_dl/downloader/common.py", line 291, in download
    return self.real_download(filename, info_dict)
  File "/usr/local/bin/youtube-dl/youtube_dl/downloader/http.py", line 66, in real_download
    data = self.ydl.urlopen(request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1244, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python2.6/urllib2.py", line 397, in open
    response = meth(req, response)
  File "/usr/lib/python2.6/urllib2.py", line 510, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.6/urllib2.py", line 435, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.6/urllib2.py", line 369, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.6/urllib2.py", line 518, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found

$ 
```

please add support for this stream

thanks

**Comment by yan12125** [comment]
youtube-dl should be working for this video in the next release. Please confirm that. If you still encounter problems, feel free to open a new issue.

**Comment by dstftw** [comment]
This has been fixed in youtube-dl 2015.04.26. See our [FAQ](https://yt-dl.org/update) if you have problems updating.
